guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -76,14 +77,76 @@ ifdef::katello[]
 # foreman-rake katello:upgrade_check
 ----
 endif::[]
+ifdef::foreman-deb[]
+. Update repositories in `/etc/apt/sources.list.d/foreman.list` on your {ProjectServer} to consume content for {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+deb http://deb.theforeman.org/ _My_Distribution_Codename_ {ProjectVersion}
+deb http://deb.theforeman.org/ plugins {ProjectVersion}
+----
++
+// List of supported OS for foreman-deb has to match "guides/common/modules/proc_configuring-repositories-foreman-deb.adoc"
+Replace _My_Distribution_Codename_ with `bullseye` if you run {ProjectServer} on Debian 11, `jammy` for Ubuntu 22.04, and `focal` for Ubuntu 20.04.
+. Update to {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+# apt-get update
+# apt-get --only-upgrade install ruby\* foreman\*
+----
+. Migrate the database on {ProjectServer}:
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake db:migrate
+# foreman-rake db:seed
+----
+. Clear the cache and existing database sessions on {ProjectServer}:
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake tmp:cache:clear
+# foreman-rake db:sessions:clear
+----
+. Optional: Reclaim space on your PostgreSQL database on {ProjectServer}:
++
+[options="nowrap" subs="attributes"]
+----
+# su - postgres -c "vacuumdb --full --dbname=foreman"
+----
+. If you have used `{foreman-installer}` to set up your existing {ProjectServer}, {Team} recommends running it again after upgrading.
+* `{foreman-installer}` can modify config files so this may overwrite your custom changes.
```

Comment:
```suggestion
. If you have used `{foreman-installer}` to set up your existing {ProjectServer}, {Team} recommends running it again after upgrading.
However, `{foreman-installer}` can modify config files so this might overwrite your custom changes.
```

Technically, I think line 119 is more tightly related to line 118 than 120. Which is why I'm suggesting this.

You could even consider using an admonition. I know that they are frowned upon in this team :) but they do exist for a reason. Your call.

```suggestion
. If you have used `{foreman-installer}` to set up your existing {ProjectServer}, {Team} recommends running it again after upgrading.
+
[IMPORTANT]
====
`{foreman-installer}` can modify config files so this might overwrite your custom changes.
====
```

Reply:
Applied your first suggestion without admonition.